ui/src/app/components/sidenav-view-assessment/sidenav-view-assessment.component.ts

```diff
@@ -7,11 +10,74 @@ import { SidenavService } from 'src/app/services/sidenav/sidenav.service'
   styleUrls: ['./sidenav-view-assessment.component.scss'],
 })
 export class SidenavViewAssessmentComponent implements OnInit {
-  constructor(private sidenav: SidenavService) {}
+  srcDbType: string = ''
+  connectionDetail: string = ''
+  conversionRateCount: ConversionRate = { good: 0, ok: 0, bad: 0 }
+  conversionRatePercentage: ConversionRate = { good: 0, ok: 0, bad: 0 }
+  badPc = 50
+  constructor(private sidenav: SidenavService, private clickEvent: ClickEventService) {}
 
-  ngOnInit(): void {}
+  dbDataSource: { title: string; source: string; destination: string }[] = []
+  dbDisplayedColumns: string[] = ['title', 'source', 'destination']
+  rateCountDataSource: { total: number; bad: number; ok: number; good: number }[] = []
+  rateCountDisplayedColumns: string[] = ['total', 'bad', 'ok', 'good']
+  ratePcDataSource: { bad: number; ok: number; good: number }[] = []
+  ratePcDisplayedColumns: string[] = ['bad', 'ok', 'good']
+
+  ngOnInit(): void {
+    this.clickEvent.viewAssesment.subscribe((data: IViewAssesmentData) => {
+      this.srcDbType = data.srcDbType
+      this.connectionDetail = data.connectionDetail
+      this.conversionRateCount = data.conversionRates
+      let tableCount: number =
+        this.conversionRateCount.good + this.conversionRateCount.ok + this.conversionRateCount.bad
+      if (tableCount > 0) {
+        for (let key in this.conversionRatePercentage) {
+          this.conversionRatePercentage[key as keyof ConversionRate] = Number(
+            ((this.conversionRateCount[key as keyof ConversionRate] / tableCount) * 100).toFixed(2)
+          )
+        }
+        console.log(this.conversionRatePercentage)
+      }
+      if (this.srcDbType != '') this.setDbDataSource()
+      if (tableCount > 0) {
+        this.setRateCountDataSource(tableCount)
+        this.setRatePcDataSource()
+      }
+    })
+  }
 
   closeSidenav() {
     this.sidenav.closeSidenav()
   }
+  setDbDataSource() {
+    this.dbDataSource = []
```

Comment:
Can we show database version as well as per figma ? : https://www.figma.com/file/mhSfts8ey0XxeHXyXdP1aC/HarbourBridge-UI?node-id=1179%3A42177